app/components/wds-global-navigation.js

```diff
@@ -71,13 +71,15 @@ export default Component.extend({
 	},
 
 	scrollHandler() {
-		const globalNavigationHeight = 55;
+		const communityBar = document.querySelector('.wds-community-bar');
```

Comment:
Could we evaluate the selector once (e.g. in didInsertElement)? If we leave it here, it will mean a DOM operation each time scroll handler is fired.